omnigibson/objects/controllable_object.py

```diff
@@ -559,7 +551,7 @@ def get_control_dict(self):
         # TODO: Move gravity force computation dummy to this class instead of BaseRobot
         fcns["gravity_force"] = lambda: (
             ControllableObjectViewAPI.get_generalized_gravity_forces(self.articulation_root_path)
-            if not self.fixed_base
```

Comment:
Is this change intended?

Reply:
yes! This was what was causing the falldown issue. we should only be using dummy values if the robot is NOT fixed base